### ISSUE TITLE
Install xiwi into existing chroot

### ISSUE THREAD

**Issue by nickbailuc** [dnschneid/crouton#3613]
```
name: stretch
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/stretch...
crouton: version 1-20171230152552~master:de1361af
release: stretch
architecture: amd64
xmethod: xorg
targets: core,gnome-desktop,xorg
host: version 10032.86.0 (Official Build) stable-channel terra 
kernel: Linux localhost 3.18.0-16288-g64d05cf80004 #1 SMP PREEMPT Mon Jan 8 23:22:23 PST 2018 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/stretch...
```
I installed Debian using the following command:

sudo sh ~/Downloads/crouton -r stretch -t core -t gnome-desktop -t xorg

I would like to add the xiwi desktop integration feature it, but do not want to reinstall from scratch because I had to navigate through many errors which disallowed the gnome-terminal to be used, and later Steam games running OpenGL, as well as allowing for an sdcard to be used via /etc/fstab. 

Is there any way to install xiwi into the existing chroot, or do I have to start over and add -t xiwi into the installation command?

If I am not using the correct terms, I mean for me to be able to switch between Debian and ChromeOS as if switching tty terminal tabs (CTRL+ALT+F1 to CTRL+ALT+F3). As of right now, I have to close everything and logout of Debian in order to access ChromeOS which is inconvenient when I am doing work and I need to change the brightness or volume, something I'm only able to do in Chrome OS.

I am using the Asus C202SA Chromebook.

**Comment by DennisLfromGA** [comment]
@nickbailuc,

> Is there any way to install xiwi into the existing chroot, or do I have to start over and add -t xiwi into the installation command?

Yes and No. ;-) 
**Yes** you can install *any* target in an existing chroot and **No**, you don't have to start over.

To add the 'xiwi' xmethod just use: 
- `sudo sh ~/Downloads/crouton -n stretch -t xiwi`

Then be sure to also install the ['crouton integration' *extension*](https://chrome.google.com/webstore/detail/crouton-integration/gcpneefbbnfalgjniomfjknbcgkbijom) from the Chrome Web Store.

Note that the above will make 'xiwi' the *default* xmethod.
You can start your desktop with either xmethod using the '-X' option like:
- For xorg: `sudo startgnome -n stretch -X xorg`
- For xiwi in a tab: `sudo startgnome -n stretch -X xiwi-tab`

> If I am not using the correct terms, I mean for me to be able to switch between Debian and ChromeOS as if switching tty terminal tabs (CTRL+ALT+F1 to CTRL+ALT+F3). 

The correct key-combo to move from CrOS to your chroot(s) and back is with the following:
- Forward: <kbd>ctrl</kbd>+<kbd>alt</kbd>+<kbd>**shift**</kbd>+<kbd>F2/forward</kbd>
- Backward: <kbd>ctrl</kbd>+<kbd>alt</kbd>+<kbd>**shift**</kbd>+<kbd>F1/back</kbd>

Note the inclusion of the <kbd>**shift**</kbd> key above.

From CrOS you can also use the 'crouton integration' app drop-down menu to switch to an active chroot.
From a chroot shell session you can move to other active chroots or CrOS with the `croutoncycle` command like:
- `croutoncycle cros`

*Hope this helps,*
-**DennisLfromGA**

**Comment by nickbailuc** [comment]
@DennisLfromGA 
I tried switching into and out of the chroot using the combination with shift, which solved the problem. Im guessing it has to be used to switch from using the Chromebook keys into the function keys, so thanks for that!

I didn't install xiwi to not potentially create more problems with the chroot, however it's very helpful that you mentioned it could be done, as I tried installing a different component into the existing chroot, prior to making this post and ended up creating an entirely new chroot, so once again thanks and I hope this helps others as well!
